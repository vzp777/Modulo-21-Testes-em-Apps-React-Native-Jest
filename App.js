import { StyleSheet, Text, View } from 'react-native';

export default function App() {
  return (
    <View style={styles.container}>
      <Text>Meu primeiro App React Native 2</Text>
     
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
