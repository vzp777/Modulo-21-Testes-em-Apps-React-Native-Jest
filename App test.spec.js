import React from "react" 
import  renderer  from 'react-test-renderer'
import App from "./App"

test('tela Ã© renderizada corretamente', () => {
    const tree = renderer.create (<App />).toJSON()
    expect(tree).toMatchSnapshot()
});