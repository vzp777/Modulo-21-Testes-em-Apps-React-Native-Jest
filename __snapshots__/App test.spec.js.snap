// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tela Ã© renderizada corretamente 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Meu primeiro App React Native 2
  </Text>
</View>
`;
